js/packages/phoenix-client/examples/apply_prompt.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-console */
+import { createClient } from "../src";
+import { getPrompt, toSDK } from "../src/prompts";
+import OpenAI from "openai";
+import { PromptLike } from "../src/types/prompts";
+
+const PROMPT_NAME = process.env.PROMPT_NAME!;
+const PROMPT_TAG = process.env.PROMPT_TAG!;
+const PROMPT_VERSION_ID = process.env.PROMPT_VERSION_ID!;
+const OPENAI_API_KEY = process.env.OPENAI_API_KEY!;
```

Comment:
I'm not sure env vars for prompt parts other than tag makes too much sense. Let's work on the examples in a followup.

Reply:
Mostly added for my own debugging. the examples here are rough and kind of hard to read, we should clean them up totally